common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3712,6 +3713,14 @@ object CelebornConf extends Logging {
       .intConf
       .createWithDefault(16)
 
+  val CLIENT_RPC_MAX_RETIRES: ConfigEntry[Int] =
+    buildConf("celeborn.client.rpc.maxRetries")
+      .categories("client")
+      .version("0.3.2")
+      .doc("Max RPC retry times in LifecycleManager.")
+      .intConf
+      .createWithDefault(2)
```

Comment:
```suggestion
      .createWithDefault(3)
```
The default value of most `maxRetries` in `CelebornConf` is 3.